internal/resource/http.go

```diff
@@ -86,6 +86,12 @@ func (f *Fetcher) UpdateHttpTimeoutsAndCAs(timeouts types.Timeouts, cas []types.
 	f.client.transport.ResponseHeaderTimeout = time.Duration(responseHeader) * time.Second
 	f.client.client.Transport = f.client.transport
 
+	// Update proxy
```

Comment:
Move this change in with the previous commit

Reply:
Change addressed in latest rebase.